src/smart-components/user/users-list-not-selectable.tsx

```diff
@@ -136,7 +137,7 @@ const UsersListNotSelectable = ({ userLinks, usesMetaInURL, props }: UsersListNo
         isCompact={false}
         borders={false}
         columns={columns}
-        rows={createRows(userLinks, users, intl)}
+        rows={createRows(userLinks, users, intl, undefined, undefined, advancedPermissions)}
```

Comment:
Let's rename it to `authModel`
```suggestion
        rows={createRows(userLinks, users, intl, undefined, undefined, authModel)}
```

Reply:
Fixed